Traceback (most recent call last):
  File "/home/jiajiexu/DT_ECO/MOSAC/DT_ECO.py", line 41, in <module>
    next_obs, rewards, terminated, truncated, infos = env.step(actions)
                                                      ^^^^^^^^^^^^^^^^^
  File "/home/jiajiexu/anaconda3/lib/python3.11/site-packages/gymnasium/wrappers/order_enforcing.py", line 56, in step
    return self.env.step(action)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/jiajiexu/anaconda3/lib/python3.11/site-packages/mo_gymnasium/envs/dt_eco/dt_eco.py", line 295, in step
    self._process_gate_features()
  File "/home/jiajiexu/anaconda3/lib/python3.11/site-packages/mo_gymnasium/envs/dt_eco/dt_eco.py", line 165, in _process_gate_features
    ef_index.append(self.graph.edge_ids(current_gate + '/' + inpin, current_gate + '/' + outpin, etype='cellarc'))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/jiajiexu/anaconda3/lib/python3.11/site-packages/dgl/heterograph.py", line 3245, in edge_ids
    u = utils.prepare_tensor(self, u, "u")
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/jiajiexu/anaconda3/lib/python3.11/site-packages/dgl/utils/checks.py", line 47, in prepare_tensor
    data = F.tensor(data)
           ^^^^^^^^^^^^^^
  File "/home/jiajiexu/anaconda3/lib/python3.11/site-packages/dgl/backend/pytorch/tensor.py", line 53, in tensor
    return th.as_tensor(data, dtype=dtype)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: new(): invalid data type 'str'
